internal/auth0/organization.go

```diff
@@ -0,0 +1,32 @@
+//go:generate mockgen -source=organization.go -destination=organization_mock.go -package=auth0
```

Comment:
could you remove this and the following too? we're not using these mocks, and it would be nice to run make generate at any momento without unwanted changes
![image](https://user-images.githubusercontent.com/11925502/124646591-3b23c900-de6b-11eb-8c98-f21feaff09a7.png)

Reply:
Done in https://github.com/auth0/auth0-cli/pull/325/commits/edcbfb815b3b20bb2e846ac51f0735583629a1a1

Reply:
thanks 🙇🏼